build.sh

```diff
@@ -5,10 +5,15 @@ set -euxo pipefail
 OS_VERSION="$1"; shift
 ARCHITECTURE="$1"; shift
 
-packer build \
-  -var os_version="$OS_VERSION" \
-  -var-file "var_files/common.pkrvars.hcl" \
-  -var-file "var_files/$ARCHITECTURE.pkrvars.hcl" \
-  -var-file "var_files/$OS_VERSION/$ARCHITECTURE.pkrvars.hcl" \
-  "$@" \
-  openbsd.pkr.hcl
+flags=(
+  "-var"      "os_version=$OS_VERSION"
+  "-var-file" "var_files/common.pkrvars.hcl"
+  "-var-file" "var_files/$ARCHITECTURE.pkrvars.hcl"
```

Comment:
Does the quoting need to be changed at all, wasn't it fine before?

Reply:
[Nope](https://github.com/cross-platform-actions/openbsd-builder/pull/19#discussion_r1730087963):

> This does not seem to work. This causes -var os_version to be passed as a single flag, when it should be passed as two.

I tested this locally with `curl`:

```sh
$ flags=("--user-agent bob")
$ curl "${flags[@]}" example.com
curl: option --user-agent bob: is unknown

$ flags=("--user-agent" "bob")
$ curl "${flags[@]}" example.com
200 OK
```